docker/images/kumascript/Dockerfile

```diff
@@ -1,44 +1,107 @@
-FROM debian:jessie
+FROM python:2.7-slim
```

Comment:
kumascript also python? I think it can use `node:6.12-slim` docker image

Reply:
Yes, I think that's a better fit for the kumascript image, as it seems well-supported, thanks @safwanrahman. I'll change it in another PR.